src/cli/onefuzz/backend.py

```diff
@@ -331,15 +334,13 @@ def config_params(
         endpoint_params = responses.Config.parse_obj(response.json())
 
         # Will override values in storage w/ provided values for SP use
```

Comment:
I don't understand your question. These checks allow us to override values supplied per command. We check each one because we don't usually want to override all of them. We only override the ones that have pre-existing value.

Reply:
We check the values b/c, for example, the client_id might be set if the CLI is being called with at SP.